tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml

```
@@ -0,0 +1,25 @@
+[setting.scbbtc_issuance_ratio]
```

we  could compact all these settings into 
```suggestion
[var.scbbtc_synth_config]
```

---

yeah i misunderstood what you meant here, but now i do.... this should not have been committed 😅

---

will fix these on sepolia too

---

hopefully resolved, please let me know 🙏🏼